### ISSUE TITLE
tools that paginate show spurious `less --version` output

### ISSUE THREAD

**Issue by Habbie** [ansible/ansible#15685]
##### ISSUE TYPE
- Bug Report
##### ANSIBLE VERSION

```
ansible 2.0.2.0
  config file =
  configured module search path = Default w/o overrides
```
##### CONFIGURATION

no configuration present/necessary
##### OS / ENVIRONMENT

`Darwin marconi.local 14.5.0 Darwin Kernel Version 14.5.0: Mon Jan 11 18:48:35 PST 2016; root:xnu-2782.50.2~1/RELEASE_X86_64 x86_64` with ansible installed via brew
##### SUMMARY

in `ansible/cli/__init__.py` there is this line:

```
        elif subprocess.call('(less --version) 2> /dev/null', shell = True) == 0:
```

but on every system I have tested, the output of `--version` goes to standard out, not standard error. It looks like the `2>` should just be `>`.
##### STEPS TO REPRODUCE

```
$ ansible-galaxy search json
less 418
Copyright (C) 1984-2007 Mark Nudelman

less comes with NO WARRANTY, to the extent permitted by law.
For information about the terms of redistribution,
see the file named README in the less distribution.
Homepage: http://www.greenwoodsoftware.com/less

Found 2 roles matching your search:

 Name                            Description
 ----                            -----------
 dereulenspiegel.alfred-json     Installs alfred-json
 minderaswcraft.chef_solo_runner Runs chef-solo giving the cookbooks archive, the json attributes (usually called node.json) file and optionaly
```

```
$ ansible-doc json
 [WARNING]: module json not found in /usr/local/Cellar/ansible/2.0.2.0/libexec/lib/python2.7/site-
packages/ansible/modules:[TRUNCATED LIST OF PATHS]

less 418
Copyright (C) 1984-2007 Mark Nudelman

less comes with NO WARRANTY, to the extent permitted by law.
For information about the terms of redistribution,
see the file named README in the less distribution.
Homepage: http://www.greenwoodsoftware.com/less
```
##### EXPECTED RESULTS

Exactly what happened, minus the output of `less --version`
##### ACTUAL RESULTS

The output includes the output of `less --version`

**Comment by selivan** [comment]
Bug reproduces on my system too, ansible 2.1.1 on Ubuntu 14.04 Trusty. Like in #17137, if you do `export PAGER=less`, bug disappears.

**Comment by bcoca** [comment]
closing via merged #17449
